src/rp2_common/hardware_clocks/include/hardware/clocks.h

```diff
@@ -287,7 +294,7 @@ bool clock_configure(clock_handle_t clock, uint32_t src, uint32_t auxsrc, uint32
  */
 void clock_configure_undivided(clock_handle_t clock, uint32_t src, uint32_t auxsrc, uint32_t src_freq);
 
-/*! \brief Configure the specified clock to use the undividded input source
+/*! \brief Configure the specified clock to use the undivided input source
```

Comment:
Hmmm, same 'brief' description for both `clock_configure_undivided` and `clock_configure_int_divider` - is that correct, or was it a copy'n'paste error?